pennylane/labs/trotter_error/product_formulas/error.py

```diff
@@ -394,6 +433,28 @@ def _compute_expectation_track_order(
     return state.dot(new_state), len(commutator)
 
 
+def _compute_expectation_single_op(
+    op: Tuple[Fragment],
+    coeff: complex,
+    order: int,
+    fragments: Dict[Hashable, Fragment],
+    state: AbstractState,
+) -> Tuple[complex, int]:
+    """Returns the expectation value obtained from a single operator/state multiplication"""
+    new_state = state
+    for symbol in op:
+        if isinstance(symbol, FrozenSet):
+            fragment = sum(
+                (frag_coeff * fragments[x] for x, frag_coeff in symbol), _AdditiveIdentity()
+            )
+        else:
+            fragment = fragments[symbol]
+
+        new_state = fragment.apply(new_state)
```

Comment:
Should we not change this to make it compute the expectation value? Or how should we manage this? This is related to the point of adding expectation values rather than states.